packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -32,6 +32,7 @@ const packageJsonPath = `${rootDir}/package.json`;
 const manifest = readManifest(manifestPath);
 const pluginArchiveFilePath = path.resolve(publishDir, `${manifest.id}.jpl`);
 const pluginInfoFilePath = path.resolve(publishDir, `${manifest.id}.json`);
+const allPossibleCategories = ['Appearance', 'Developer tools', 'Productivity', 'Themes', 'Integrations', 'Note management', 'Search', 'Tags', 'Editor', 'Files'];
```

Comment:
this whole list should be lowercase probably

Reply:
Done :+1: